.github/workflows/ci-cd.yml

```
@@ -35,3 +35,7 @@ jobs:
       - name: Test
         shell: bash
         run: yarn test
+
+      - name: Generate templates
```

minor: should we write:

```suggestion
      - name: Generate and Build Templates
```